### DISCUSSION TITLE
GitHub Education Pack Missing from my account

### DISCUSSION THREAD

**Discussion by nexospro** [community/community#131866] [GitHub Education] [answered]
Hi everyone,

I've noticed that my GitHub Education Pack is no longer available after my github account diactived. Previously, I had access to various tools and resources through the Education Pack, but it seems to have disappeared recently.

Has anyone else experienced this issue? If so, were you able to resolve it? Any advice on how to get the Education Pack restored or reasons why it might have been removed would be very helpful.

Thank you for your assistance!

**Comment by nexoscreator** [comment, marked as answer]
Hi,

I faced a similar issue recently. Here are a few steps you can take to try and resolve it:

1. Check Your Eligibility: Ensure that you still meet the eligibility criteria for the GitHub Education Pack. Sometimes, changes in your educational status can affect access.

2. Reapply: If your access was removed, try reapplying for the [GitHub Education Pack](https://education.github.com/pack) here. Make sure to provide all necessary documentation.

3. Contact GitHub Support: Reach out to [GitHub Support](https://support.github.com/ ) directly through their contact form. Provide them with details about your situation and any relevant documentation.

**Reply by nexospro** [reply]
> Hi,
> 
> I faced a similar issue recently. Here are a few steps you can take to try and resolve it:
> 
> 1. Check Your Eligibility: Ensure that you still meet the eligibility criteria for the GitHub Education Pack. Sometimes, changes in your educational status can affect access.
> 2. Reapply: If your access was removed, try reapplying for the [GitHub Education Pack](https://education.github.com/pack) here. Make sure to provide all necessary documentation.
> 3. Contact GitHub Support: Reach out to [GitHub Support](https://support.github.com/) directly through their contact form. Provide them with details about your situation and any relevant documentation.

Let's me check out this

**Comment by nexospro** [comment]
Okay, let's me check out
